ci-operator/step-registry/rosa/cluster/provision/rosa-cluster-provision-commands.sh

```diff
@@ -500,8 +500,7 @@ if [[ ${ENABLE_SHARED_VPC} == "yes" ]]; then
   echo "    SAHRED_VPC_BASE_DOMAIN: ${SAHRED_VPC_BASE_DOMAIN}"
 fi
 
-echo -e "
-rosa create cluster -y \
+echo -e "rosa create cluster -y \
```

Comment:
Do we need to update this?

Reply:
we may not anymore. But may be better to avoid the newline at the beginning of the file. wdyt ?

Reply:
No need. Both are ok for me.